helm/values.yaml

```
@@ -0,0 +1,66 @@
+# Default values for immudb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: codenotary/immudb
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+adminPassword: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+   capabilities:
+     drop:
+     - ALL
+   runAsNonRoot: true
+   runAsUser: 3322
+   runAsGroup: 3322
+
+service:
+  type: ClusterIP
+  ports:
+    grpc: 3322
+    metrics: 9497
+    http: 8080
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hostname: immudb-example.localhost
```

The `.local` TLD suffix is reserved for this kind of domains, maybe better to use:

```suggestion
  hostname: immudb-example.local
```

---

Actually, RFC [2606](https://datatracker.ietf.org/doc/html/rfc2606) reserve domains .test, .example, .invalid and **.localhost** for demostrative use. In many resolver implementation (I use unbound) every domain with TLD .localhost resolve to 127.0.0.1. That's why I've chosen that.

**.local** is used by the bonjour protocol and only resolves to the local lan.